fx-src/index.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const Cu = Components.utils;
+
+import BaseAdapter from "../src/adapters/base";
+import KintoBase from "../src/KintoBase";
+import FirefoxAdapter from "./FirefoxStorage";
+
+export default function loadKinto() {
+  let { EventEmitter } = Cu.import("resource://gre/modules/devtools/shared/event-emitter.js", {});
```

Comment:
because I initially wrote it in a scratchpad (where const gets annoying) and forgot to change it! Will fix